src/store/reducers/connectionsCache/connectionsCache.ts

```diff
@@ -12,26 +16,98 @@ const connectionsCacheSlice = createSlice({
       state,
       action: PayloadAction<ConnectionShortDetails[]>
     ) => {
-      state.connections = action.payload;
+      if (!Array.isArray(action.payload) || !action.payload.length) return;
```

Comment:
I think we can remove this line and just let the error throw, as this shouldn't happen. Otherwise, it's becomes harder to find the regression when we introduce a bug